doc/tutorial/didi/carprice_xgboost/README.md

```diff
@@ -91,7 +96,8 @@ LABEL msrp
 INTO sqlflow_models.my_xgb_regression_model;
 ```
 
-# Predict the car's price
+## Predict the Car Price
+
 After training the regression model, let's predict the car price using the trained model.
 
 First, we can specify the trained model by USING clause:
```

Comment:
And using a standar SQL to fetch the prediction data:   standar-->standard; 
Than, we can specify the prediction result table by PREDICT clause:
 : than-->then

Reply:
thanks, done.